jalib/jalloc.cpp

```diff
@@ -400,6 +404,31 @@ jalib::JAllocDispatcher::deallocate(void *ptr, size_t)
 {
   ::free(ptr);
 }
+
```

Comment:
As elsewhere, we may no longer need this commit.

If the code below is needed, we seem to be adding it for all CPUs.
Maybe that truly is useful for debugging.  Now that time has passed, please review, and decide if this still seems useful.